YoCode/Program.cs

```diff
@@ -33,21 +33,10 @@ private static async Task Main(string[] args)
             var compositeOutput = new Output(new CompositeWriter(outputs), outputPath, (IErrorReporter)outputs.Find(a => a is ConsoleWriter));
 
             var appSettingsBuilder = new AppSettingsBuilder(result.JuniorTest);
-            var parameters = new RunParameterChecker(compositeOutput, result, appSettingsBuilder);
 
-            if (!parameters.ParametersAreValid())
-            {
-                if (!result.HelpAsked)
-                {
-                    compositeOutput.ShowInputErrors(parameters.Errs);
-                }
-                else
-                {
-                    compositeOutput.ShowHelp();
-                    LaunchReport(result, outputPath);
-                }
-                return;
-            }
+            var parameters = new RunParameterChecker(compositeOutput, appSettingsBuilder);
+
+            parameters.ParametersAreValid(outputPath, result);
```

Comment:
This function looks weird if you're not checking the return value. It should do what it did before - check the return value and then take appropriate action. `ParametersAreValid` doesn't communicate the severity of what's going to happen.